.github/actions/update-axe-core-v1/src/getPackageManager.test.ts

```diff
@@ -0,0 +1,49 @@
+import 'mocha'
```

Comment:
Where are these 3rd party libraries coming from? There's nothing specified in the package manifest.